packages/flutter/test/widgets/slivers_test.dart

```diff
@@ -196,6 +227,58 @@ void main() {
     expect(find.text('BOTTOM'), findsOneWidget);
   });
 
+  testWidgets('SliverFixedExtentList correctly clear garbage', (WidgetTester tester) async {
+    final List<String> items = <String>['1', '2', '3', '4', '5', '6'];
+    await testSliverFixedExtentList(tester, items);
+    TestGesture gesture = await tester.startGesture(const Offset(400.0, 500.0));
+    await gesture.moveBy(const Offset(0.0, -400.0));
+    await tester.pump();
```

Comment:
Out of curiousity, why not just use `tester.drag`?

Reply:
fixed